tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -208,8 +233,8 @@ ROOT::Experimental::Detail::RPageSinkDaos::CommitClusterGroupImpl(unsigned char
                                          RNTupleCompressor::MakeMemCopyWriter(bufPageListZip.get()));
 
    auto offsetData = fOid.fetch_add(1);
```

Comment:
TBD: maybe use a different counter for cluster page lists?

Reply:
Could be! In fact, `fOid` just became a misnomer with this PR, since it's counting committed pages but not necessarily unique object IDs anymore, depending on the mapping.